### DISCUSSION TITLE
Can you add login functionality?

### DISCUSSION THREAD

**Discussion by amdefiguy** [mikeyhodl/radiofront#2] [Q&A] [answered]
Hi,
Thanks for this, I see great potential in the project. Can you add login functionality to the application?

**Comment by mikeyhodl** [comment, marked as answer]
The login functionality is in progress I t will be live in the coming days. Thanks for the concern

**Comment by amdefiguy** [comment]
Can I run the project using `next dev` ?

**Comment by mikeyhodl** [comment]
Currently you cannot, use 

`npm i` to install packages
`npm run dev` to start the development environment
